package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "alphagov-wordpress-theme",
+  "version": "1.0.0",
+  "description": "Alpha Gov WordPress Theme",
+  "author": "govwordpress@gov.bc.ca",
+  "license": "Apache-2.0",
+  "keywords": [
+    "WordPress",
+    "Block Theme",
+    "BCGov",
+    "AlphaGov",
+    "Gutenberg"
+  ],
+  "homepage": "https://github.com/bcgov/alphagov-wordpress-theme#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bcgov/alphagov-wordpress-theme.git"
+  },
+  "main": "index.js",
+  "scripts": {
+    "build": "wp-scripts build src/scripts/public/font.js --output-path=dist",
+    "build:production": "npm ci && npm run fix:js && npm run lint:js && npm run fix:css && npm run lint:css && npm run lint:md:docs && npm run test:unit && npm run build",
+    "lint:css": "wp-scripts lint-style src/styles",
+    "fix:css": "wp-scripts lint-style --fix --config @wordpress/stylelint-config/scss src/styles",
+    "lint:js": "wp-scripts lint-js src/scripts",
+    "fix:js": "wp-scripts lint-js src/scripts --fix",
+    "format": "wp-scripts format src/",
+    "lint:md:docs": "wp-scripts lint-md-docs --config .markdownlint.json --ignore-path .markdownlintignore",
```

Comment:
I added the ignore and the config so that it wouldn't lint the checklist. If running `composer checklist` overwrites the current checklist, then the user would have to edit the datestamp on the first line of checklist.md every time a new checklist is run. I can test this though.

Reply:
Ah I see. I misread it as ignoring the changelog but it's ignoring the checklist which makes sense. We can keep the ignore of the checklist here since it will fail the linting every time otherwise but we should probably fix the checklist output in wordpress-scripts to produce valid markdown and then we don't have to worry about it.